Update distributed tracing digests and pipelines

andreasgerstmayr opened the pull request:
Update distributed tracing digests to latest versions.

andreasgerstmayr committed:
Update distributed tracing digests

Signed-off-by: Andreas Gerstmayr <agerstmayr@redhat.com>

jgbernalp commented:
/ok-to-test

jgbernalp commented:
@andreasgerstmayr it seems the tekton cel expression might be missing the right values as the pipeline was not started. Can you check?

andreasgerstmayr committed:
fix CEL expressions for tracing UI

The source is included as a Git submodule, i.e. the GitHub API returns a
single file change, therefore we need to update to remove the wildcard
in the CEL expression.

Signed-off-by: Andreas Gerstmayr <agerstmayr@redhat.com>

andreasgerstmayr commented:
> @andreasgerstmayr it seems the tekton cel expression might be missing the right values as the pipeline was not started. Can you check?

should be fixed now

falox commented on .tekton/distributed-tracing-console-plugin-0-3-1-2-pull-request.yaml:
```diff
@@ -13,7 +13,7 @@ metadata:
               ".tekton/distributed-tracing-console-plugin-0-3-1-2-push.yaml".pathChanged() ||
               "Dockerfile.ui-distributed-tracing-pf4".pathChanged() ||
-              "ui-distributed-tracing-pf4/***".pathChanged())
+              "ui-distributed-tracing-pf4".pathChanged())
```
hey @andreasgerstmayr , are these changes generated by customize-pipeline.sh?

falox commented:
/ok-to-test

andreasgerstmayr committed:
update customize-pipeline.sh script

Signed-off-by: Andreas Gerstmayr <agerstmayr@redhat.com>

andreasgerstmayr committed:
fix Dockerfile.ui-distributed-tracing-pf5 and related tekton pipeline

Signed-off-by: Andreas Gerstmayr <agerstmayr@redhat.com>

andreasgerstmayr commented:
I noticed some copy&paste errors in the dockerfile, I fixed them in this PR also.

jgbernalp commented:
/ok-to-test
